miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/permission/MiniAppCustomPermissionCache.kt

```diff
@@ -17,24 +17,60 @@ internal class MiniAppCustomPermissionCache(context: Context) {
         "com.rakuten.tech.mobile.miniapp.custom.permissions.cache", Context.MODE_PRIVATE
     )
 
+    @VisibleForTesting
+    fun isDataExist(miniAppId: String) = prefs.contains(miniAppId)
```

Comment:
Maybe this sounds more correct:
```suggestion
    fun doesDataExist(miniAppId: String) = prefs.contains(miniAppId)
```